interlok-core/src/main/java/com/adaptris/core/ConfigurableEventHandler.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2015 Adaptris Ltd.
```

Comment:
Do we still say we're Adaptris? Also note the year.

Reply:
@domule I can update the year but what is the correct legal entity to put here?